src/core/public/saved_objects/simple_saved_object.ts

```diff
@@ -77,9 +80,14 @@ export class SimpleSavedObject<T = unknown> {
 
   public save(): Promise<SimpleSavedObject<T>> {
     if (this.id) {
-      return this.client.update(this.type, this.id, this.attributes, {
-        references: this.references,
-      });
+      return this.client
+        .update(this.type, this.id, this.attributes, {
+          references: this.references,
+        })
+        .then((sso) => {
+          this.updatedAt = sso.updatedAt;
+          return sso;
+        });
     } else {
       return this.client.create(this.type, this.attributes, {
         migrationVersion: this.migrationVersion,
```

Comment:
I think you also need to update `updateAt` when `create`-ing the object?

Reply:
fixed and included a test. woops!